recipes/tinkerforge-bindings/all/conanfile.py

```diff
@@ -70,8 +70,7 @@ def package(self):
         cmake.install()
 
     def package_info(self):
-        self.cpp_info.set_property("cmake_target_name", "tinkerforge-bindings")
-        self.cpp_info.set_property("cmake_find_mode", "both")
+        self.cpp_info.set_property("cmake_target_name", "tinkerforge::bindings")
```

Comment:
I am having issues with the variable names generated for this target with cmake 3.17.1 & conan 1.42.1:

```
tinkerforge-bindings/2.1.32 (test package): Calling build()
-- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.19044.
-- The C compiler identification is MSVC 19.29.30038.1
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.29.30037/bin/Hostx64/x64/cl.exe
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.29.30037/bin/Hostx64/x64/cl.exe - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Conan: called by CMake conan helper
-- Conan: Adjusting output directories
-- Conan: Using cmake targets configuration
-- Library tinkerforge_bindings found C:/Users/danimtb/.conan/data/tinkerforge-bindings/2.1.32/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/tinkerforge_bindings.lib
-- Conan: Adjusting default RPATHs Conan policies
-- Conan: Adjusting language standard
-- This project seems to be plain C, using 'MSVC' compiler
-- Conan: Using autogenerated Findtinkerforge-bindings.cmake
-- Found tinkerforge-bindings: 2.1.32 (found version "2.1.32")
CMake Error at build/ecbfb49a4e2ea8618319be5f9e02c8e0621391fb/Findtinkerforge-bindings.cmake:125 (conan_find_apple_frameworks):
  Syntax error in cmake code at

    C:/Users/danimtb/repos/conan-center-index/recipes/tinkerforge-bindings/all/test_package/build/ecbfb49a4e2ea8618319be5f9e02c8e0621391fb/Findtinkerforge-bindings.cmake:125

  when parsing string

    ${tinkerforge::bindings_FRAMEWORKS}

  Invalid character (':') in a variable name: 'tinkerforge'
Call Stack (most recent call first):
  CMakeLists.txt:7 (find_package)


CMake Error at build/ecbfb49a4e2ea8618319be5f9e02c8e0621391fb/Findtinkerforge-bindings.cmake:125 (conan_find_apple_frameworks):
  Syntax error in cmake code at

    C:/Users/danimtb/repos/conan-center-index/recipes/tinkerforge-bindings/all/test_package/build/ecbfb49a4e2ea8618319be5f9e02c8e0621391fb/Findtinkerforge-bindings.cmake:125

  when parsing string

    ${tinkerforge::bindings_FRAMEWORK_DIRS}

  Invalid character (':') in a variable name: 'tinkerforge'
Call Stack (most recent call first):
  CMakeLists.txt:7 (find_package)
```

Reply:
I think that's because you need to set min required version to 1.43.0

Reply:
Added. Thanks!